tests/OSPSuite.HelpersForTests/ModuleHelperForSpecs.cs

```diff
@@ -72,12 +74,16 @@ public SimulationConfiguration CreateSimulationConfigurationForExtendMergeBehavi
          var module2 = createModule2();
 
          module2.MergeBehavior = MergeBehavior.Extend;
+         //TODO Add back when event issue is fixed
+         // module1.Add(createEventGroupBuildingBlock("EventForModule1", "eventGroup1", "eventBuilder1"));
+         // module2.Add(createEventGroupBuildingBlock("EventForModule2", "eventGroup2", "eventBuilder1"));
+
          simulationConfiguration.AddModuleConfiguration(new ModuleConfiguration(module1));
-         // Using this constructor to validate that the merge behavior is set correctly
          simulationConfiguration.AddModuleConfiguration(new ModuleConfiguration(module2, null, null));
          return simulationConfiguration;
       }
 
+      //TODO REMOVE. This method is called extend and override at the same time!
```

Comment:
I don't think it should be removed as per the TODO. The naming is off probably due to this method being introduced when we were able to override the module behavior when creating the simulation configuration. We removed that since.

I think that's the intended meaning of `Override` in the method name, the simulation configuration was overriding the module default behaviour.

You can just rename it. It's now simply a simulation configuration with an extend module.